LRUCache.py

```diff
@@ -0,0 +1,22 @@
+TC : O(1)
```

Comment:
I would recommend you to implement the hash map + linked list approach where you code out evrything